lib/ABI/FunctionType.cpp

```diff
@@ -150,12 +150,6 @@ class ConversionHelper {
     for (auto &SavedRegister : D.CalleeSavedRegisters)
```

Comment:
Commit message: what do you mean by "root conversions"?

Reply:
Sorry, `conversion roots`. As in, only the main body of the algorithm, not touching the details hidden away in helpers.

I will reword this one.

Reply:
I reworded it last week, but forgot to respond here 😅.

[ABI: reorder and annotate the conversion roots](https://github.com/revng/revng/pull/232/commits/7258d11194524c45830014fa62fa95c1b5a25efc)
``` 
Since further down the line, some extensive changes are made to these
conversion root functions, the reordering is kept separate to improve
how diffs look for the mentioned commits.

This enforces the fact that return values types need to be taken care
of before the arguments since they can impact how the arguments get
distributed among the registers.
```

Reply:
What do you think?